src/Switch/Switch.spec.js

```diff
@@ -41,7 +48,8 @@ describe('<Switch />', () => {
   });
 
   it('should spread custom props on the SwitchBase node', () => {
```

Comment:
This would be a consistency problem amongst the selection controls.  Structurally, switch is slightly different from Radio and Checkbox.  All structurally are different without a label.  From an intent, the `div` around the switch is merely there as an implementation detail to provide the background when it is `on`, so I think it is appropriate and consistent in these cases to spread unused at the `SwitchBase`